### ISSUE TITLE
preload images?

### ISSUE THREAD

**Issue by mllewis** [langcog/website#13]
On slow internet connects, the pictures take awhile to load

**Comment by mcfrank** [comment]
on the people page, you mean? I guess one way to deal with that is to preload them on the splash page? but otherwise, we could also decrease their size.

**Comment by mllewis** [comment]
Yeah, on the people page. Pre-loading on the splash page seems like it
would solve the problem.

Molly
